src/sqlancer/questdb/test/QuestDBQueryPartitioningBase.java

```diff
@@ -66,12 +67,15 @@ protected ExpressionGenerator<Node<QuestDBExpression>> getGen() {
     @Override
     public void check() throws SQLException {
         s = state.getSchema();
-        targetTables = s.getRandomTableNonEmptyTables();
```

Comment:
Can we also remove the corresponding field, or is it still used anywhere?

Reply:
Well, I can delete it. But I'm considering we'll implement support for multiple tables (e.g. `Join`) in the future, so I think it's not a big deal to delete it probably?

Reply:
But anyway, if it's better to delete it now, I'll do that in this PR :)

Reply:
Deleted and committed.